compiler/qsc_eval/src/tests.rs

```diff
@@ -127,7 +127,7 @@ fn check_partial_eval_stmt(
     let std_fir = qsc_lowerer::Lowerer::new().lower_package(&std.package);
     let std_id = store.insert(std);
 
-    let sources = SourceMap::new([("test".into(), file.into())], Some(expr.into()));
+    let sources = SourceMap::new([("".into(), file.into())], Some(expr.into()));
```

Comment:
Is this required to get the test passing? I think I'd rather we update the test case to accommodate the file name `test`, rather than set this slightly dangerous precedence for blank file names.